symfony/client/src/orangehrmAttendancePlugin/components/RecordCell.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div v-show="date" class="oxd-table-card-cell">
+    <div v-show="showHeader" class="header">
+      {{ header.title }}
+    </div>
+    <div class="data">
+      <oxd-text tag="p">
+        {{ date }} {{ time }}
+        <oxd-text tag="span" class="timezone"> GMT {{ offset }} </oxd-text>
```

Comment:
can you check, do we need to localize `GMT {{ offset }}`

Reply:
I checked in existing lang strings it's not there. will check with Kavinda.

Reply:
Kavinda agreed GMT is not needed to localise. (makes sense since it's an abbreviation) 
![image](https://user-images.githubusercontent.com/12880059/155299338-6134149c-fe0c-4d60-97d3-d12c699f8114.png)